### PR TITLE
Handle serializable strategy config change

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: scalardb-test/src/main/java/kelpie/scalardb/Common.java [hunk 1]
```diff
@@ -101,7 +101,8 @@ public static DatabaseConfig getDatabaseConfig(Config config) {
     props.setProperty(DatabaseConfig.NAMESPACE_PREFIX, prefix);
     props.setProperty(DatabaseConfig.TRANSACTION_MANAGER, transactionManager);
     props.setProperty(DatabaseConfig.ISOLATION_LEVEL, isolationLevel);
-    props.setProperty(DatabaseConfig.SERIALIZABLE_STRATEGY, serializableStrategy);
+//    props.setProperty(ConsensusCommitConfig.SERIALIZABLE_STRATEGY, serializableStrategy);
```
Comment (feeblefakie) [inline scalardb-test/src/main/java/kelpie/scalardb/Common.java:104]: We can't use this one?

**Review by feeblefakie — COMMENTED** [review]
Thank you! Left one question.

File: scalardb-test/src/main/java/kelpie/scalardb/Common.java (hunk 1 above)
Comment (brfrn169) [reply scalardb-test/src/main/java/kelpie/scalardb/Common.java:104]: Thank you for reviewing this!

> We can't use this one?

Yes for now. `ConsensusCommitConfig` will be introduced in the next release (3.2.0). So I will use it after 3.2.0 is released.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**